content/docs/05.developer-guide/02.tasks.md

```diff
@@ -253,6 +253,53 @@ You cannot access the output of a sibling task as tasks will be run in parallel.
 ::
 
 
+### Dag
+
+This task allows defining dependencies between tasks by creating a directed acyclic graph (DAG) without bothering with the graph structure. List your tasks and their dependencies, and Kestra will figure out the rest.
```

Comment:
```suggestion
This task allows defining dependencies between tasks by creating a directed acyclic graph (DAG). Instead of an explicit DAG structure, this task allows you to only define upstream dependencies for each task using the `dependsOn` keyword. This way, you can set dependencies more implicitly for each task, and Kestra will figure out the overall flow structure.
```

Reply:
I prefere yours ;) 
I took this one from the DAG task description.
One small nit: it's dependsOn is a property not a keyword, I'll update later.